examples/site/testing_pubsub/pubsub_integration_test.cc

```diff
@@ -180,10 +181,17 @@ bool PubsubIntegrationTest::WaitForServerReady(std::string const& url) {
         "message": { "data": "" }
       }
     })js";
-    auto r = HttpEvent(url, kPing);
-    auto line = NextChildLine();
-    if (r.code == kOkay) return true;
-    std::cerr << "... [" << r.code << "] " << line << std::endl;
+    try {
+      auto const r = HttpEvent(url, kPing);
+      auto line = NextChildLine();
+      if (r.code == kOkay) return true;
+      std::cerr << "... [" << r.code << "] " << line << std::endl;
```

Comment:
Observation: The other ones with an `r.code` don't log here.  Should they, or should this not?

Reply:
Thanks for noting the inconsistency, the logs do help troubleshoot things, added everywhere.